unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -55,13 +58,19 @@ public class BuilderInWorldController : MonoBehaviour
     [HideInInspector]
     public bool isBuilderInWorldActivated = false;
 
+    private const float RAYCAST_MAX_DISTANCE = 10000f;
+    private const string LAND_EDITION_NOT_ALLOWED_BY_PERMISSIONS_MESSAGE = "This land does not belong to you, nor have you been granted operating permits by its owner.";
+    private const string LAND_EDITION_NOT_ALLOWED_BY_SDK_LIMITATION_MESSAGE = "This place was created with the SDK and can not be edited in-world.";
+    private const float CACHE_TIME_LAND = 5 * 60;
+    private const float CACHE_TIME_SCENES = 1 * 60;
+    private const float REFRESH_LANDS_WITH_ACCESS_INTERVAL = 2 * 60;
```

Comment:
Should this be moved to the settings? At least the cache, this way we will have all the settings in the same place

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/563/commits/d8f45c487609117aed1f93f4a885e3c9fba323ce. Thanks!